README.md

```diff
@@ -30,15 +30,26 @@ https://gitlab.kitware.com/cmake/community/wikis/FAQ#out-of-source-build-trees
 
 CMake is available with `port install cmake` or `brew install cmake` (on OS X, with MacPorts or Homebrew respectively), or with `aptitude install cmake` (on debian).
 
-To compile the development version from this repository, do:
+To compile and install the development version from this repository, do:
 
 	git clone https://github.com/MesserLab/SLiM.git
 	mkdir SLiM_build
 	cd SLiM_build
 	cmake -D CMAKE_BUILD_TYPE=Release ../SLiM
 	make
+    make install
 
-The resulting binary will be called `slim`, within the `SLiM_build` directory.  You can name the build directory anything you wish; there is nothing magical about the name `SLiM_build`.
+The `make install` step will install the `slim` and `eidos` binaries to appropriate system directories (hopefully).
+If you don't want this to happen, just skip this step; the binaries
+will be located within the `SLiM_build` directory.
+You can name the build directory anything you wish; there is nothing magical about the name `SLiM_build`.
+To install the binaries to a different directory (like the `--user` flag), instead run
+
+	cmake -D CMAKE_BUILD_TYPE=Release -D -DCMAKE_INSTALL_PREFIX:PATH=/path/to/install ../SLiM
```

Comment:
here's how to tell it where you want to install

Reply:
How about "but this will probably fail because of permissions unless you run it with root permissions (i.e., `sudo make install`).  You can instead tell cmake..."

I think we ought to explicitly point people toward using sudo.  On an HPC cluster or some such, they may not have the ability to do that (but might want to ask an admin to do it for them), but on a user's local machine that is what they will probably want to do, so we should make the procedure explicit.

Reply:
I've adjusted this. How is it now?

Reply:
Looks good, thanks.  I'll accept these pull requests eventually; I'm not waiting for anything in particular, but I'm busy being on vacation, and there's no urgency to them as far as I'm aware of them, so I'll deal with them later.  If anybody is impatient for them to go in sooner, let me know.  :->